WordPress/WordPressHomeWidgetToday/Views/Localization/LocalizedStringKey+extension.swift

```diff
@@ -0,0 +1,12 @@
+import SwiftUI
+
+
+extension LocalizedStringKey {
+    static let defaultBundle = Bundle(for: HomeWidgetTodayRemoteService.self)
+
+    /// TODO - TODAYWIDGET - LocalizedStringKey is used as a wrapper of NSLocalizedString, in order to use synthetic keys and assign a default value
```

Comment:
It might not be worth mentioning this as a `TODO` just because IMHO it's unlikely to change any time soon. It seems like this was a great (missed) opportunity for Apple to clean this up and have `LocalizedStringKey` have proper fallback behaviour and they didn't, so I wouldn't expect this to change.

I like the comment though explaining _why_ it exists and the circumstances under which it could be replaced. Just not sure if `TODO` is that useful since it might be in that state for a looooooong time 😄